packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -166,6 +171,16 @@ class TestCommand extends FastFlutterCommand {
             'Test files must be in that directory and end with the pattern "_test.dart".'
         );
       }
+    } else {
+      final List<String> fileCopy = <String>[];
+      for (String file in files) {
+        if (file.endsWith(platform.pathSeparator)) {
```

Comment:
This appears to have broken the IntelliJ test identification for directories.  Is it tractable to identify what is and isn't a directory.  Is it possible to check whether the file system entity for this path is a directory?

Reply:
we can check if the entity is a directory instead of using the path check. I'll open a PR to do so

> This appears to have broken the IntelliJ test identification for directories.

I would strongly suggest writing a test with the behavior that intellij expects so I don't accidentally break you again